<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectorSettingsDialog</class>
 <widget class="QDialog" name="DetectorSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DetectorSettingsDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,2,2,2">
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0,0,0" rowminimumheight="0,0,0,0,0,0,0,0">
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="bSpinboxQ1">
       <property name="minimum">
        <double>-127.000000000000000</double>
       </property>
       <property name="maximum">
        <double>128.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Brightness: </string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Brightness: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QDoubleSpinBox" name="cSpinboxQ1">
       <property name="minimum">
        <double>-127.000000000000000</double>
       </property>
       <property name="maximum">
        <double>128.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Contrast:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Segment Q2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Brightness: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Segment Q1</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Brightness: </string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="bSpinboxQ2">
       <property name="minimum">
        <double>-127.000000000000000</double>
       </property>
       <property name="maximum">
        <double>128.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="bSpinboxQ3">
       <property name="minimum">
        <double>-127.000000000000000</double>
       </property>
       <property name="maximum">
        <double>128.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="bSpinboxQ4">
       <property name="minimum">
        <double>-127.000000000000000</double>
       </property>
       <property name="maximum">
        <double>128.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QDoubleSpinBox" name="cSpinboxQ2">
       <property name="minimum">
        <double>-127.000000000000000</double>
       </property>
       <property name="maximum">
        <double>128.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QDoubleSpinBox" name="cSpinboxQ3">
       <property name="minimum">
        <double>-127.000000000000000</double>
       </property>
       <property name="maximum">
        <double>128.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QDoubleSpinBox" name="cSpinboxQ4">
       <property name="minimum">
        <double>-127.000000000000000</double>
       </property>
       <property name="maximum">
        <double>128.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Segment Q3</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Segment Q4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Contrast:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Contrast:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Contrast:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Detector port angle:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="angleSpinbox">
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Detector settings</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
